### ISSUE TITLE
Timeout issue

### ISSUE THREAD

**Issue by ruzickap** [tcort/markdown-link-check#72]
Sometimes I'm getting the false positive from `markdown-link-check` when checking the "slow" URLs:

```bash
$ cat > /tmp/links << EOF
https://techcrunch.com/
EOF

$ markdown-link-check --verbose /tmp/links

FILE: /tmp/links
[✖] https://techcrunch.com/ → Status: 0 Error: ETIMEDOUT

ERROR: dead links found!
```

Is there a way how I can increase the timeout for checking some slow URLs?

Thank you...

**Comment by sschuberth** [comment]
Same here e.g. for https://source.android.com/setup/develop/repo.

**Comment by NicolasMassart** [comment]
Hi, there's no way to change the default timeout right now. Thanks for raising this issue. We'll look at making this happen.

**Comment by NicolasMassart** [comment]
will be fixed by #124
